<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="LI5650_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="LI5650Auto_checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>120</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650Voltage_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage (V)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LI5650Voltage_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650Frequency_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency (Hz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LI5650Frequency_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650SignalInput_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Input</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LI5650SignalInput_comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="LI5650FilterSlope_comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>6 dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18 dB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24 dB</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LI5650FilterSlope_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter Slope</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LI5650</string>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650ReferenceSignal_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference Signal</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LI5650ReferenceSignal_comboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>REF IN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INT OSC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SIGNAL</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="LI5650TimeConstant_lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650TimeConstant_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Constant (s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LI5650Connect_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LI5650IfRun_checkBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="LI5650Address_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LI5650Address_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="SR830_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>561</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="SR830Address_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SR830Address_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="SR830Frequency_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency (Hz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SR830Frequency_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SR830_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SR830</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SR830Connect_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SR830IfRun_checkBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SR830Voltage_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage (V)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SR830Voltage_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SR830Phase_lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="other_frame">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>560</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="other_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>other</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otherWaitingTime_lineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>33</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="otherWaitingTime_label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>41</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wait After Setting Voltage (s):</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>470</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_pushButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>470</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QFrame" name="N5244A_frame">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>561</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="N5244A_label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="N5244AAddress_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="N5244A_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>N5244A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="N5244AConnect_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="N5244AIfRun_checkBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="N5244ASweepType_comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>CW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Power</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frequence</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="N5244A_label_14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep Type</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="N5244ASweepType_tab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="N5244ACW_tab">
     <attribute name="title">
      <string>CW</string>
     </attribute>
     <widget class="QLineEdit" name="N5244ACW_frequence_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="N5244ACW_powerOn_checkBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>35</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244ACW_power_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CW Frequence (GHz)</string>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Power (dBm):</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="N5244APower_tab">
     <attribute name="title">
      <string>Power</string>
     </attribute>
     <widget class="QLabel" name="N5244A_label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Power(dB)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244APower_startPower_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244APower_stopPower_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_19">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>75</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Power (dB)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244APower_points_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244APower_frequence_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_22">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequence</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="N5244AFrequence_tab">
     <attribute name="title">
      <string>Frequence</string>
     </attribute>
     <widget class="QLabel" name="N5244A_label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Points</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244AFrequence_points_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244AFrequence_startFrequence_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Frequence (GHz)</string>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_16">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>75</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Frequence (GHz)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244AFrequence_stopFrequence_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="N5244AFrequence_powerOn_checkBox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>35</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="N5244AFrequence_power_lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>35</y>
        <width>140</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="N5244A_label_18">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Power (dBm):</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="N5244AAverage_checkBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Average</string>
    </property>
   </widget>
   <widget class="QLabel" name="N5244A_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Averaging Factor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="N5244AAveragingFactor_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="N5244AIF_BW_lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="N5244A_label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IF BW (Hz)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LI5650IfRun_checkBox</tabstop>
  <tabstop>LI5650Connect_pushButton</tabstop>
  <tabstop>LI5650Address_lineEdit</tabstop>
  <tabstop>LI5650ReferenceSignal_comboBox</tabstop>
  <tabstop>LI5650Auto_checkBox</tabstop>
  <tabstop>LI5650Voltage_lineEdit</tabstop>
  <tabstop>LI5650FilterSlope_comboBox</tabstop>
  <tabstop>LI5650SignalInput_comboBox</tabstop>
  <tabstop>LI5650Frequency_lineEdit</tabstop>
  <tabstop>LI5650TimeConstant_lineEdit</tabstop>
  <tabstop>SR830IfRun_checkBox</tabstop>
  <tabstop>SR830Connect_pushButton</tabstop>
  <tabstop>SR830Voltage_lineEdit</tabstop>
  <tabstop>SR830Address_lineEdit</tabstop>
  <tabstop>SR830Frequency_lineEdit</tabstop>
  <tabstop>SR830Phase_lineEdit</tabstop>
  <tabstop>N5244AIfRun_checkBox</tabstop>
  <tabstop>N5244AConnect_pushButton</tabstop>
  <tabstop>N5244AAddress_lineEdit</tabstop>
  <tabstop>N5244AIF_BW_lineEdit</tabstop>
  <tabstop>N5244AAveragingFactor_lineEdit</tabstop>
  <tabstop>N5244AAverage_checkBox</tabstop>
  <tabstop>N5244ASweepType_comboBox</tabstop>
  <tabstop>N5244ASweepType_tab</tabstop>
  <tabstop>N5244ACW_power_lineEdit</tabstop>
  <tabstop>N5244ACW_frequence_lineEdit</tabstop>
  <tabstop>N5244ACW_powerOn_checkBox</tabstop>
  <tabstop>N5244APower_points_lineEdit</tabstop>
  <tabstop>N5244APower_startPower_lineEdit</tabstop>
  <tabstop>N5244APower_stopPower_lineEdit</tabstop>
  <tabstop>N5244AFrequence_points_lineEdit</tabstop>
  <tabstop>N5244AFrequence_power_lineEdit</tabstop>
  <tabstop>N5244AFrequence_powerOn_checkBox</tabstop>
  <tabstop>N5244AFrequence_startFrequence_lineEdit</tabstop>
  <tabstop>N5244AFrequence_stopFrequence_lineEdit</tabstop>
  <tabstop>otherWaitingTime_lineEdit</tabstop>
  <tabstop>apply_pushButton</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
